diff --git setup.py setup.py
index 6cd5c20..c985c9c 100644
--- setup.py
+++ setup.py
@@ -4,7 +4,6 @@ from setuptools.command.sdist import sdist as _sdist
 
 # Partially adapted from https://github.com/OP2/PyOP2/blob/master/setup.py
 
-
 pyx_sources = [op.join('cyassimp', 'cyassimpwrapper.pyx')]
 cythonized_sources = [op.join('cyassimp', 'cyassimpwrapper.cpp')]
 external_sources = [op.join('cyassimp', 'cpp', 'assimpwrapper.cpp')]
@@ -15,6 +14,28 @@ ext_kwargs = {
     'language': 'c++'
 }
 
+# This is a patch for windows with Conda that allows easy installation of
+# cyassimp on Windows! We essentially just copy the DLLs into the path
+# so that compiling/run time linking works.
+import sys
+import os
+import shutil
+from glob import glob
+
+def localpath(*args):
+    return os.path.abspath(reduce(os.path.join, (os.path.dirname(__file__),) + args))
+
+
+if sys.platform.startswith('win'):
+    CONDA_ASSIMP_DIR = os.environ['CONDA_ASSIMP_DIR']
+    if CONDA_ASSIMP_DIR is not None:
+        ext_kwargs['include_dirs'] = [os.path.join(CONDA_ASSIMP_DIR, 'include')]
+        ext_kwargs['library_dirs'] = [os.path.join(CONDA_ASSIMP_DIR, 'lib')]
+        dlls = glob(os.path.join(CONDA_ASSIMP_DIR, 'lib', 'Assimp*.dll'))
+        for d in dlls:
+            basename = os.path.basename(d)
+            shutil.copy(d, localpath('cyassimp', basename))
+
 ext_name = 'cyassimp.cyassimpwrapper'
 
 
@@ -93,7 +114,7 @@ setup(name='cyassimp',
       ],
       ext_modules=extensions,
       packages=find_packages(),
-      package_data={'cyassimp': ['*.pyx', 'cpp/*.h']},
+      package_data={'cyassimp': ['*.pyx', 'cpp/*.h', '*.dll']},
       cmdclass=cmdclass,
       setup_requires=['numpy>=1.8.0'],
       install_requires=['numpy>=1.8.0']
